test/integration/offchain/attachment.test.ts

```diff
@@ -121,4 +123,12 @@ describe('attachment', () => {
       'content-disposition': 'attachment; filename="json"',
     })
   })
+
+
+    it('Doesn`t upload files if more than 100mb', async () => {
+      const uploadRes = await postFile(app, '/attachment', Buffer.from(overSizeBlobData), 'json')
+      const { status  } = await get(app, `/attachment/${uploadRes.body.id}`)
```

Comment:
FYI: const { status, body ....

Reply:
But surely with this I only need the response to prove that the file has not been uploaded?